examples/express-api-reference/src/index.ts

```diff
@@ -49,6 +49,7 @@ app.use(
 )
 
 // Listen
-app.listen(5055, () => {
-  console.log('💻 Express listening on http://localhost:5055')
+const PORT = 8080
```

Comment:
This is a bad port to use. Very common in old webpack apps

Reply:
For deployment this should likely be loaded by a .env file and not hardcoded

Reply:
Generally I agree but is this really necessary for a small example app?

Reply:
Yeah let's just leave the port alone and in the .env file. This is in alignment with all of our other projects in both repos.